packages/calypso-products/src/sort.js

```diff
@@ -1,14 +1,15 @@
-/**
- * External dependencies
- */
-
-import { difference, flatten, groupBy, sortBy } from 'lodash';
-
 /**
  * Internal dependencies
  */
 import { getDomainProductRanking, isCredits, isDomainProduct, isPlan } from '.';
 
+// https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore
+const groupBy = ( items, key ) =>
+	items.reduce(
+		( r, v, i, a, k = v[ key ] ) => ( ( r[ k ] || ( r[ k ] = [] ) ).push( v ), r ),
```

Comment:
Would be great if we could make this a tad more readable 😉  The part where we declare the array if it doesn't exist may require a bit more eyeballing to read. What do you think?

Reply:
Yeah, this was not ideal. I've removed the file, though, so that should resolve it 😂